metrics.md

```diff
@@ -0,0 +1,57 @@
+gluster_brick_inodes_free | gluster_brick_inodes_free {volume="xxx", brick_path="/mnt/xxx", peerid="xxxxx", host="xxxx"} 100	|
+| gluster_brick_inodes_total |	gluster_brick_inodes_total{volume="xxx", brick_path="/mnt/xxx", peerid="xxxxx", host="xxx"} 200 |
+| gluster_brick_inodes_available | gluster_brick_inodes_available{volume="xxx", brick_path="/mnt/xxx", peerid="xxxxx", host="xxx"} 100 |
+| gluster_brick_capacity_total | gluster_brick_capacity_total{volume="xxx", brick_path="xxxx", peerid="xxx", host="xxx"} 1048576	|
+| gluster_brick_capacity_used|	gluster_brick_capacity_used{volume="xxx", brick_path="xxxx", peerid="xxx", host="xxx"} 1048	|
+| gluster_brick_capacity_free |	gluster_brick_capacity_free{volume="xxx", brick_path="xxx", peerid="xxx", host="xxx"} 1047576	|
+ | gluster_disk_read_ios | gluster_disk_read_ios{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+ |   gluster_disk_read_merges | gluster_disk_read_merges{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+ |   gluster_disk_read_sectors | gluster_disk_read_sectors{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+ |   gluster_disk_read_ticks | gluster_disk_read_ticks{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+ |   gluster_disk_write_ios | gluster_disk_write_ios{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+ |   gluster_disk_write_merges | gluster_disk_write_merges{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+ |   gluster_disk_write_sectors | gluster_disk_write_sectors {brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+ |   gluster_disk_write_ticks | gluster_disk_write_ticks {brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+ |   gluster_disk_inflight | gluster_disk_inflight{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+ |   gluster_disk_total_ticks | gluster_disk_total_ticks{brick_path="xxx", device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+  |  gluster_disk_time_inqueue | gluster_disk_time_inqueue{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+
+#### Gluster process Metrics: 
+
+|Gluster Prometheus schema |example| comments |
+|--------|-----------|------|
+| gluster_virtual_memory | gluster_virtual_memory{name="glusterfsd",nodeid="xxx",brick_path="/bricks/b1",volume="gv1", host="xxx"} 912260
```

Comment:
@vredara My concern is related to metrics that are process-level statistics, such as `gluster_virtual_memory` and others in this section of the doc.
The example exposes a `brick_path` for the process, but this same process may have many different brick paths when mux is enabled. If an admin were to sum(`gluster_virtual_memory`) across all volumes or bricks on a node, they would get a number that is unrealistic because the same process would be counted multiple times. Likewise, providing virtual memory usage and tagging it with a specific volume or brick path is not correct because *that* volume may not be responsible for the vast majority of the memory usage... it could be any one of the (n-1) other bricks that are mux-ed into the same glusterfsd.